src/bindings/python/tests/test_transformations/test_pattern_ops.py

```diff
@@ -20,7 +20,7 @@
 
 
 def test_wrap_type_pattern_type():
-    last_opstet_number = 10
+    last_opstet_number = 11
```

Comment:
last_opstet_number => last_opset_number

Reply:
I suppose also [similar](https://github.com/openvinotoolkit/openvino/pull/15238/files) changes are required?

Reply:
There's no implementation of the new TopK (and no downgrading transformation) so we should not return this op anywhere yet. The changes you linked to will be done in the scope of 101500.